amundsen_application/static/js/ducks/notification/api/tests/index.spec.ts

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
+
+import { sendNotification } from '../v0';
```

Comment:
nit: We're using a new pattern here to make imports nicer for files that will have a lot of imports. This one doesn't but we should be consistent.

`import * as API from '../v0'`

and then call

`API.sendNotification(...)`